src/main/java/mate/academy/springboot/swagger/util/ParserUtil.java

```diff
@@ -0,0 +1,34 @@
+package mate.academy.springboot.swagger.util;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.springframework.data.domain.Sort;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ParserUtil {
+    private static final String SEPARATOR1 = ":";
+    private static final String SEPARATOR2 = ";";
+
+    public List<Sort.Order> getOrders(String sortBy) {
+        List<Sort.Order> orders = new ArrayList<>();
+        if (sortBy.contains(SEPARATOR1)) {
```

Comment:
There should be a check for two separators, now we can``t use order like this `orderBy=name;price`